### PR TITLE
Add pruner task for pipeline runs

### PR DESCRIPTION
**Pull request by lkolacek**
CLOUDBLD-7635

Signed-off-by: Ladislav Kolacek <lkolacek@redhat.com>

### CODE REVIEW

**Comment by lkolacek** [comment]
pipelinerun pruner completed after adding fix https://console-openshift-console.apps.mpp-w2-preprod.cfln.p1.openshiftapps.com/k8s/ns/exd-sp-guild-container-build--osbs-qa-source-containers/jobs/pipelinerun-cleaner-1635762000/

I'll take a look also on pod pruning

**Comment by lkolacek** [comment]
@chmeliik @ben-alkov @MartinBasti I fixed pod pruner as well. Please take a look.

File: tasks/configure_namespace.yml [hunk 1]
```diff
@@ -104,6 +104,32 @@
...
+        schedule: "*/5 * * * *"
+        jobTemplate:
+          spec:
+            template:
+              spec:
+                serviceAccountName: "{{ osbs_pruner_sa }}"
+                containers:
+                  - name: tkn
+                    image: registry.redhat.io/openshift-pipelines/pipelines-cli-tkn-rhel8:0.19.0-2
+                    command: ["tkn", "pipelinerun", "delete"]
```
[the remark this answers is not in the document]
Comment (lkolacek) [reply tasks/configure_namespace.yml]: great thanks ben. Resolving comment.

File: tasks/configure_namespace.yml [hunk 2]
```diff
@@ -137,40 +137,38 @@
...
       metadata:
         name: "{{ item.name }}"
-        namespace: "{{ osbs_ocp_namespace }}"
       spec:
-        backoffLimit: 6
-        completions: 1
-        parallelism: 1
-        template:
+        # hourly
+        schedule: "* */1 * * *"
```
Comment (chmeliik) [inline tasks/configure_namespace.yml:151]: I pasted this to https://crontab.guru and it says "At every minute past every hour", as in "every minute"

Comment (lkolacek) [reply tasks/configure_namespace.yml]: Nice catch Adam .. useful tool! I changed it to `0 * * * *` hope this will reflect hourly.

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Pull request merged**